CHANGES.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+### 0.1.7 2017-01-03
+
+* Update gltf-pipeline to 0.1.0-alpha9
+* Added command to generate documentation (npm run jsdoc)
```

Comment:
How is the generated doc viewed?  Will it be deployed somewhere?

If I just open the HTML file, the image link to the Cesium logo is wrong since it is not in a subdirectory.
```html
<a href="http://cesiumjs.org/"><img src="doc/cesium.png" /></a>
```